string Class
int64 class_n
float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
